src/components/e-button.md

```diff
@@ -135,3 +135,9 @@ Creates a button like link.
 ```vue
 <e-button @click="onStyleguideClick">A button<br />with wrapping<br />text</e-button>
 ```
+
+#### Custom element
+
```

Comment:
Please add a hint, that this option should only be used, when using the button component within e.g. a html anchor tag to make sure to prevent standalone buttons being span or similiar